app/views/cart/show.html.erb

```diff
@@ -25,9 +25,19 @@
           <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}"%>
         </p>
       </td>
+      <% discount = Discount.find_by(merchant_id: item.merchant_id) %>
+      <% item_order = ItemOrder.find_by(item_id: item.id) %>
+      <% if !discount.nil? && discount.item_quantity == quantity %>
       <td>
-        <p><%=number_to_currency(item.price)%></p>
+        <p><%= number_to_currency(item.price - (item.price / discount.percentage_discount))%></p>
+        <p>Item price has been adjusted due to discount!</p>
+        <% item.price -= (item.price / discount.percentage_discount) %>
```

Comment:
Tried to set item price here for future use, but scope prevents it from being used from outside the item block.